### ISSUE TITLE
[Feature]: Add support for Hedera native staking

### ISSUE THREAD

**Issue by mgarbs** [LedgerHQ/ledger-live#2139]
### Library name

ledgerhq/app-hedera

### Describe your feature request

As a trusted and reliable provider of secure hardware wallets, Ledger is uniquely positioned to put Hedera user's minds at ease with reliable security while staking. Currently, it is only possible to stake in software wallets, though Hedera is supported by Ledger. This is a great opportunity for both communities.

Not only will this feature appeal to Hedera users looking for a secure and convenient way to stake their tokens, but it will also attract new users to the Ledger and Hedera communities. With native staking on the Ledger products, Hedera stakers can rest assured that their tokens are secure and accessible at all times, making it easier and more convenient to participate in the staking process and earn rewards.

Additionally, Hedera native staking on the Ledger products will drive more adoption of web3 technologies and help to further promote the growth and development for all of web3. With Ledger's reputation for security and reliability, users can trust that their coins are being staked safely and securely, further enhancing the appeal of the Hedera network and driving more users to Ledger to participate in the native staking process. Hedera offers a highly attractive solution for staking where users earn without locking up their coins and no slashing penalties for unstaking.

In short, adding native Hedera staking to the Ledger products will not only bring new users to Ledger, but it will also help drive the adoption of web3 as a whole by promoting the growth of decentralization by keeping the keys with the users. I believe this feature would be a valuable addition to the Ledger community and greatly benefit web3 alltogether.
